File: hackertracker/src/main/java/com/shortstack/hackertracker/Home/FAQRenderer.java
package com.shortstack.hackertracker.Home;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.pedrogomez.renderers.Renderer;
import com.shortstack.hackertracker.R;

import java.util.List;

import butterknife.Bind;
import butterknife.ButterKnife;

public class FAQRenderer extends Renderer<String[]> {

    public static final int POS_QUESTION = 0;
    public static final int POS_ANSWER = 1;

    @Bind(R.id.question)
    TextView question;

    @Bind(R.id.answer)
    TextView answer;

    @Override
    protected View inflate(LayoutInflater inflater, ViewGroup parent) {
        View view = inflater.inflate(R.layout.row_faq, parent, false);
        ButterKnife.bind(this, view);
        return view;
    }

    @Override
    public void render(List<Object> payloads) {
        question.setText(getContent()[POS_QUESTION]);
        answer.setText(getContent()[POS_ANSWER]);
    }
}

File: hackertracker/src/main/java/com/shortstack/hackertracker/Renderer/FAQRenderer.kt
package com.shortstack.hackertracker.Renderer

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.pedrogomez.renderers.Renderer
import com.shortstack.hackertracker.Model.FAQ
import com.shortstack.hackertracker.R
import kotlinx.android.synthetic.main.row_faq.view.*

class FAQRenderer : Renderer<FAQ>() {

    override fun inflate(inflater: LayoutInflater, parent: ViewGroup): View {
        return inflater.inflate(R.layout.row_faq, parent, false)
    }

    override fun render(payloads: List<Any>) {
        rootView.question.text = content.question
        rootView.answer.text = content.answer
    }
}
